File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.java
class Solution {
    public String solution(String s) {
        String answer = "";
        String[] strs = s.split(" ",-1); //"3people", "unFollowed, "me"	
              StringBuilder sb = new StringBuilder();          
        
        for (int i = 0; i < strs.length; i++) {
            String st = strs[i];
            
         if (!st.isEmpty()) { // 공백이 아닌 경우만 변환

            String first = st.substring(0,1);
            String remain = st.substring(1);
            
            String result = first.toUpperCase() + remain.toLowerCase() ;
            sb.append(result);
               
         }
            if (i < strs.length - 1) {
                sb.append(" ");
            }
        }
          answer = sb.toString();
        return answer;
    }
}

File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.js
function solution(s) {
    // 문자열 분리
    const words = s.split(" ")
    
    const wordsLower = words.map(x=> {
        // 첫글자는 대문자
        if (x === "") return ;
        const first = x[0].toUpperCase();
        const rest = x.slice(1).toLowerCase();
        
        return first+rest;
    });
   
    return wordsLower.join(" ")
}